teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Smina is a command line tool, which currently does not <strong>provide</strong> a Python API</li><li>Beside<strong>s</strong> providing <strong>the</strong> ligand,</li><li>setting the number of docking poses <strong>to be generated</strong></li></ul><p> </p><p>Briefly comment what "exhaustiveness" is and how it affects the calculation and results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>